app/models/admin_post.rb

```diff
@@ -79,6 +79,23 @@ def translated_post_language_must_differ
     errors.add(:translated_post_id, "cannot be same language as original post")
   end
 
+  ####################
+  # DELAYED JOBS
+  ####################
+
+  include AsyncWithResque
+  @queue = :utilities
+
+  # Turns off comments for all posts that are older than the configured time period.
+  # If the configured period is nil or less than 1 day, no action is taken.
+  def self.disable_old_post_comments
+    return unless ArchiveConfig.ADMIN_POST_COMMENTING_EXPIRATION_DAYS&.positive?
+
+    where.not(comment_permissions: :disable_all)
+      .where(created_at: ..(Time.current - ArchiveConfig.ADMIN_POST_COMMENTING_EXPIRATION_DAYS.days))
```

Comment:
Do you think it'd be clearer to use `ago` here? Or is subtracting from `Time.current` more intuitive?

Reply:
Yes (to the first one), thanks! I overthought this line a bunch and totally missed that :facepalm: